socketshark/__init__.py

```diff
@@ -243,13 +251,20 @@ def run(context, config):
     # Configure root logger if logging level is specified in config
     if log_config['level']:
         level = getattr(logging, log_config['level'])
-        logger = logging.getLogger()
-        logger.setLevel(level)
         formatter = logging.Formatter(log_config['format'])
         sh = logging.StreamHandler()
         sh.setFormatter(formatter)
+
+        logger = logging.getLogger(log_config['logger_name'])
+        logger.setLevel(level)
         logger.addHandler(sh)
 
+        if log_config['trace_level']:
+            trace_level = getattr(logging, log_config['trace_level'])
+            trace_logger = logging.getLogger(log_config['trace_logger_name'])
+            trace_logger.setLevel(trace_level)
+            trace_logger.addHandler(sh)
```

Comment:
I think this and the `addHandler(sh)` above could just be added to the root logger